contrib/babelfishpg_tsql/sql/upgrades/babelfishpg_tsql--4.4.0--4.5.0.sql

```diff
@@ -10,6 +10,45 @@ SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false)
  * So make sure that any SQL statement (DDL/DML) being added here can be executed multiple times without affecting
  * final behaviour.
  */
+
+DO $$
+DECLARE
+    exception_message text;
+BEGIN
+    ALTER FUNCTION sys.hashbytes(IN alg VARCHAR, IN data VARCHAR) RENAME TO hashbytes_varchar_deprecated_4_5_0;
```

Comment:
do we really need to deprecate this function? what if some cx objects are dependent on this and they are upgrading their database to new version (major version upgrade)?